ros_gz_image/CMakeLists.txt

```diff
@@ -16,32 +16,11 @@ find_package(ros_gz_bridge REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(sensor_msgs REQUIRED)
 
-if(NOT DEFINED ENV{GZ_VERSION})
-  set(ENV{GZ_VERSION} "garden")
-endif()
-
-# Garden
-if("$ENV{GZ_VERSION}" STREQUAL "garden")
-  find_package(gz-transport12 REQUIRED)
-  find_package(gz-msgs9 REQUIRED)
-
-  set(GZ_TARGET_PREFIX gz)
-  set(GZ_MSGS_VER ${gz-msgs9_VERSION_MAJOR})
-  set(GZ_TRANSPORT_VER ${gz-transport12_VERSION_MAJOR})
+find_package(gz_transport_vendor REQUIRED)
+find_package(gz-transport REQUIRED)
```

Comment:
It seems removing this (and `gz-msgs` below) fixes this package's build, but not sure that is the right approach.

Reply:
The issue was in the upstream gz_msg vendor package. A similar fix to https://github.com/gazebo-release/gz_utils_vendor/pull/3 has been released to all the vendor packages, but hasn't been synced to ros-testing yet. Hopefully by tomorrow.